lib/packwerk/detect_stale_deprecated_references.rb

```diff
@@ -0,0 +1,14 @@
+# typed: true
+# frozen_string_literal: true
+
+require "packwerk/cache_deprecated_references"
+
+module Packwerk
+  class DetectStaleDeprecatedReferences < CacheDeprecatedReferences
+    def stale_violations?
```

Comment:
Same as my previous comment. Can you add type annotations to this class?

Reply:
done

Reply:
🙌